master/buildbot/newsfragments/ssh-batch-mode.feature

```diff
@@ -0,0 +1,11 @@
+SSH connections are now run with ``-o BatchMode=yes`` to prevent interactive
```

Comment:
For the future, it's best to keep the release notes short, one-two sentences unless it's a change that affects majority of users in a way that requires their attention (e.g. deprecation/replacement of frequently used feature). It's best to explain this in the documentation of affected steps.